docs/README.md

```diff
@@ -2,10 +2,11 @@
 
 Welcome to the Aragon Connect documentation.
 
-* The **Guides** section contains guides that will help you build a dapp using Aragon Connect.
-* The **Advanced** section contains guides that are helpful if you want to write your own app connector, or if you want to have a deeper understanding on how things work.
-* The **Connectors** section contains in-depth documentation of the Connectors availables.
-* The **API Reference** section contains a complete documentation of the functions and objects exposed by Aragon Connect.
+* The [**Guides**](./guides) section contains guides that will help you build a dapp using Aragon Connect.
+* The [**Advanced**](./advanced) section contains guides that are helpful if you want to write your own app connector or want to gain a deeper understanding of everything under the hood.
+* The [**Connectors Reference**](./connectors-reference) section contains in-depth documentation of the [Connectors](./api-reference/connectors.md) available.
```

Comment:
I was thinking of using “Connectors” for this section: I would like each of these to also contain a “how to use it” with examples, which some of them have already.